apps/testingaccessibility/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -27,6 +27,7 @@ export default withSentry(SkillRecordings(skillOptions))
 
 export const config = {
   api: {
+    bodyParser: false,
```

Comment:
this is important for webhooks because they need raw requests and next body parsing affects that